manage_breast_screening/notifications/migrations/0022_messagebatch_silent.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 5.2.7 on 2025-11-14 09:18
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('notifications', '0021_alter_clinic_code_clinic_notificatio_code_55dbdb_idx_and_more'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='messagebatch',
+            name='silent',
```

Comment:
just a minor nitpick, would be good for boolean value to be called `is_silent` but maybe just personal preference :)